staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service_unix_test.go

```diff
@@ -153,10 +153,10 @@ func TestTimeouts(t *testing.T) {
 
 				f, err := mock.NewBase64Plugin(socketName.path)
 				if err != nil {
-					t.Fatalf("failed to construct test KMS provider server, error: %v", err)
```

Comment:
This seems more correct to me

Reply:
1) Changing from Fatalf to Errorf will miss some things. Like #100392
2) According to #92402, I will split and submit.

/close